tutorials/hello/hello_client/src/app/plugins/hello_info/db.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable consistent-return */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+import { codec, db as liskDB } from 'lisk-sdk';
+import * as os from 'os';
+import { join } from 'path';
+import { ensureDir } from 'fs-extra';
+import { newHelloEventSchema, counterSchema, heightSchema } from './schemas';
+import { Event, Counter, Height } from './types';
+import { DB_KEY_ADDRESS_INFO, DB_LAST_COUNTER_INFO, DB_LAST_HEIGHT_INFO } from './constants';
+
+const { Database } = liskDB;
+type KVStore = liskDB.Database;
+
+// Returns DB's instance.
+export const getDBInstance = async (
+    dataPath: string,
+    dbName = 'lisk-framework-helloInfo-plugin.db',
+): Promise<KVStore> => {
+    const dirPath = join(dataPath.replace('~', os.homedir()), 'plugins/data', dbName);
+    await ensureDir(dirPath);
+    return new Database(dirPath);
+};
+
+// Returns event's data stored in the database.
+export const getEventHelloInfo = async (db: KVStore, lastCounter: number): Promise<Event> => {
+    try {
+        let dbKey = Buffer.from(lastCounter.toString());
+        dbKey = Buffer.concat([dbKey, Buffer.from(':', 'utf8'), DB_KEY_ADDRESS_INFO]);
+        const encodedAddressInfo = await db.get(dbKey);
+        return codec.decode<Event>(newHelloEventSchema, encodedAddressInfo);
+    } catch (error) {
+        return error;
+    }
+};
+
+// Stores event's data in the database.
+export const setEventHelloInfo = async (db: KVStore, _lskAddress: Buffer, _message: string, _eventHeight: number, lastCounter: number): Promise<Event> => {
+    try {
+        const encodedAddressInfo = codec.encode(newHelloEventSchema, { senderAddress: _lskAddress, message: _message, height: _eventHeight });
+        let dbKey = Buffer.from(lastCounter.toString());
```

Comment:
you can use `intToBuffer(lastCounter, 4)` from cryptography library instead of converting to string